ui/src/common/components/RangeAggregation/RangeAggregation.jsx

```diff
@@ -133,12 +144,40 @@ function getSliderMarks([lower, upper], [min, max]) {
   };
 }
 
+function getHintTitles(namespace) {
+  switch (namespace) {
+    case DATA_NS:
+    case AUTHOR_DATA_NS:
+      return {
+        selectedTitle: 'Selected Datasets',
+        totalTitle: 'Total Datasets',
+      };
+    case AUTHOR_PUBLICATIONS_NS:
+    case AUTHOR_CITATIONS_NS:
+    case CONFERENCE_CONTRIBUTIONS_NS:
+    case EXPERIMENT_PAPERS_NS:
+    case INSTITUTION_PAPERS_NS:
+    case JOURNAL_PAPERS_NS:
+    case LITERATURE_NS:
+      return {
+        selectedTitle: 'Selected Papers',
+        totalTitle: 'Total Papers',
+      };
+    default:
+      return {
+        selectedTitle: 'Selected Papers',
+        totalTitle: 'Total Papers',
+      };
```

Comment:
```suggestion
    case LITERATURE_NS:
    default:
      return {
        selectedTitle: 'Selected Papers',
        totalTitle: 'Total Papers',
      };
```

Reply:
Also we could add https://github.com/inspirehep/inspirehep/blob/78fbb7fd6f865d46b50e0405833ba80bc24c9826/ui/src/common/utils.ts#L181

Reply:
@drjova updated! For the pluralize, I am not single if it makes sense in this particular case. for the `totalTitle` for sure no. Could be for the selected one ?